acceptance/config/beaker/options.rb

```diff
@@ -8,6 +8,7 @@
   "puppetservice" => 'puppetserver',
   "use-service" => true,
   "master-start-curl-retries" => 60,
-  "puppetpath" => "/etc/puppetlabs/agent"
+  "puppetpath" => "/etc/puppetlabs/puppet",
+  "puppetserver-confdir" => '/etc/puppetlabs/puppetserver/conf.d'
```

Comment:
About the `puppetserver-confdir` being in beaker or not, we're in a unique situation that we want beaker to be able to introspect the host for that value, but we don't have the puppetserver equivalent of `puppet master --configprint ...` to actually do that. 

However, the setting is essentially global (as in beaker doesn't support multiple masters), so it's relatively easy for a project (like puppet) to add that to `options.rb`. And since this change only affects `aio` type acceptance tests, we can make this change in beaker without breaking existing consumers, e.g. puppet#stable.